nixos/modules/services/torrent/transmission.nix

```
@@ -251,6 +251,16 @@ in
           For instance, SSH sessions may time out more easily.
         '';
       };
+
+      webHome = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        example = "pkgs.flood-for-transmission";
+        description = lib.mdDoc ''
+          Used as the value of `TRANSMISSION_WEB_HOME` if not `null`.
```

Here's a bit better and more informative documentation:

```suggestion
          If not `null`, sets the value of the `TRANSMISSION_WEB_HOME`
          environment variable used by the service. Useful for overriding
          the web interface files, without overriding the transmission
          package and thus requiring rebuilding it locally. Use this if
          you want to use the web interface provided by
          `pkgs.flood-for-transmission`.
```

Perhaps you could even modify the last sentence suggested above to write exactly how to use the package with this option? I mean, is it simply:

```nix
services.transmission.webHome = pkgs.flood-for-transmission;
```

Or is it:

```nix
services.transmission.webHome = "${pkgs.flood-for-transmission}/public";
```

Or something alike?

---

Thanks! I followed your suggestion for the most part. Regarding the usage of this option, the first example is enough.